test/parallel/test-whatwg-url-origin.js

```diff
@@ -42,7 +45,7 @@ function runURLTests(urltests) {
     var expected = urltests[i]
     if (typeof expected === "string" || !("origin" in expected)) continue
     test(function() {
-      var url = bURL(expected.input, expected.base)
+      var url = bURL(expected.input || expected.url, expected.base)
```

Comment:
Also these new data can be upstreamed to [WPT](https://github.com/w3c/web-platform-tests/tree/master/url), no need to block this PR though, we can remove the duplicates in `url-tests-additional.js` if it's upstreamed and  synced into `url-tests.json` later.